common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -177,7 +177,8 @@ export interface ISequencedDocumentAugmentedMessage extends ISequencedDocumentMe
 }
 
 export interface ISignalMessage {
-
+    // TODO: Update this to use undefined instead of null.
```

Comment:
You may not be able to do this depending on how entrenched the existing API is. Protocol defs is pretty low-level so permeates the stack.